ARTstor.js

```diff
@@ -1,22 +1,25 @@
 {
-	"translatorID": "5278b20c-7c2c-4599-a785-12198ea648bf",
-	"label": "ARTstor",
-	"creator": "Sebastian Karcher",
-	"target": "^https?://library\\.artstor\\.org",
-	"minVersion": "3.1",
-	"maxVersion": "",
```

Comment:
The URLs are:
www.sscommons.org/openlibrary
library.artstor.org/library
sahara.artstor.org/library
openlibrary.artstor.org/openlibrary